book_source/04_advanced_user_guide/02_adding_to_pecan/01_case_studies/02_add_formats.Rmd

```diff
@@ -0,0 +1,127 @@
+# Formats records in BETY {#NewFormat}
+
+The PEcAn database keeps track of all the input files passed to models, as well as any data used in model validation or data assimilation. Before we start to register these files with PEcAn we need to define the format these files will be in.
+
+The main goal is to take all the meta-data we have about a data file and create a record of it that pecan can use as a guide when parsing the data file. 
+
+This information is stored in a Format record in the bety database. Make sure to read through the current Formats before deciding to make a new one. 
+
+# Creating a new Format record in BETY
+
+If the Format you are looking for is not available, you will need to create a new record. Before entering information into the database, you need to be able to answer the following questions about your data:
+
+- What is the file MIME type? We have a suit of functions for loading in data in open formats such as CSV, txt, netCDF, etc. 
+     - PEcAn has partnered with the [NCSA BrownDog project](http://browndog.ncsa.illinois.edu/) to create a service that can read and convert as many data formats as possible. 
+     - If your file type is less common or a proprietary type, you can use the [BrownDog DAP](http://dap.ncsa.illinois.edu/) to convert? 
+     - If BrownDog cannot convert your data, you will need to contact us about writing a data specific load function.
+  
+- What variables does the file contain?
+     - What are the variables named?
+     - What are the variable units?
+     - How do the variable names and units map to PEcAn variables?
```

Comment:
In the future you might provide a link to a more detailed explanation of how one goes about figuring this out. When I've done it, it's required a lot of searching the Variables table for any conceivably related variable names and keywords. Might also be a good place to remind folks to NOT create redundant variables (same meaning but different name or different units) as that will break the ability to automate model/data and data/data comparisons